docs/README.md

```diff
@@ -8,11 +8,28 @@ All content reflects the current state of discussion, not final decisions.
 
 # Eclipse Dataspace Connector
 
-TBD
+EDC is an interoperable, cross-organization framework for data sharing between 2 or more Organizations in a trusted and
+compliant way. This framework has focused to develop in 4 key technological key points for safer data sharing: Identity
+of the user(s), Trust, Policies and Interoperability. The Eclipse Dataspace Connector framework provides different
+modules for performing data queries, exchanges, policy enforcement, monitoring and auditing.
```

Comment:
Not necessary as it does not pertain to the domain model

Reply:
Yes, you are right, I should have created a different pull request for this one. This readme document is located inside the "docs" folder and I thought it was a good idea to fill it a bit since it just had TBDs in every title.